main/options/options.cc

```diff
@@ -295,6 +295,7 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                        stop_after_options, ", ", [](const auto &pr) -> auto{ return pr.option; }));
 
     // Advanced options
+    options.add_options("advanced")("rbi", "Parse an inline RBI string", cxxopts::value<string>()->default_value(empty.inlineRBIInput), "string");
```

Comment:
I think I might call the option `--e-rbi`, I think that makes it maybe a little less likely people misinterpret it? e.g., I could see someone mistake this flag as a zero-argument flag that forces Sorbet to treat all `*.rb` files after it as RBI files, or something else.

Reply:
Sure, updated now.